src/bindings/python/flux/wrapper.py

```diff
@@ -5,6 +5,9 @@
 import cffi
 from types import MethodType
 
+import traceback
```

Comment:
Thanks for the catch, I had that in there to print stack traces while debugging a free before use issue. Will post an update later today.




On September 14, 2016 at 7:04:27 AM PDT, Stephen Herbein <notifications@github.com> wrote:

In src/bindings/python/flux/wrapper.py<https://github.com/flux-framework/flux-core/pull/807#discussion_r78754406>:

> @@ -5,6 +5,9 @@
>  import cffi
>  from types import MethodType
>
> +import traceback


unused import?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/flux-framework/flux-core/pull/807/files/a3ec52d781e81d9b994dc209383c11117c8bac05#r78754406>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AAoStXEakYDxD32pDWO7DU1XZhB3q_mmks5qp_7igaJpZM4J8O6W>.